packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -54,16 +62,29 @@ class PoolMaster extends Component {
     gpuGroups: createGetObjectsOfType('gpuGroup')
       .filter((_, { pool }) => ({ $pool: pool.id }))
       .sort(),
+    migrationNetwork: createGetObject((_, { pool }) => pool.otherConfig['xo:migrationNetwork']),
+    pifsWithIps: createGetObjectsOfType('PIF')
+      .pick((_, { networks }) => flatMap(networks, 'PIFs'))
```

Comment:
Do you need to do this?

Reply:
Ok, but I think it would make more sense to first get the pool's PIFs in [`pool/index.js`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/pool/index.js#L37-L47) and then filter them here to only get the ones that have an IP. It would give you the exact same PIFs, right?